docs/howto/add_link.md

```diff
@@ -0,0 +1,336 @@
+[Back to Guides](../README.md)
+
+# How to add links
+
+`ActiveModelSerializers` offers you many ways to add links in your JSON reponse depending on the adapter you are using.
+
+Note that within the following examples, we set the `adapter` option, but that is not needed if the `ActiveModelSerializers.config.adapter` variable is properly set. Also, some examples are given for both Rails and [non-Rails usage](outside_controller_use.md).
+
+Unless stated otherwise, the examples given here will assume that the following classes are defined:
+
+```ruby
+class User < ActiveModelSerializers::Model
+  attributes :id, :name
+end
+
+class UserSerializer < ActiveModel::Serializer
+  attributes :id, :name
+end
+```
+
+## JSON API adapter
+
+The JSON API specification allows the usage of the `links` member for representing links in three cases:
+* [Top level links](http://jsonapi.org/format/#document-top-level)
+* [Resource link](http://jsonapi.org/format/#document-resource-object-links)
+* [Relationship links](http://jsonapi.org/format/#document-resource-object-relationships)
+
+`ActiveModelSerializers` provides tools to handle these links.
+
+### [Top level links](http://jsonapi.org/format/#document-top-level)
+
+In order to provide top-level links, you will need to specify the `links` option to the serialization:
+
+```ruby
+user = User.new(id: 1, name: "John")
+links = { home: "https://www.example.com/" }
```

Comment:
I sort of feel if they don't know how to use `render` this isn't the place to learn it